docs/_static/css/custom.css

```diff
@@ -1,3 +1,25 @@
+h1 {
+  /*color: #D30074;*/
```

Comment:
Why commented? Maybe can be deleted?

Reply:
It is for the new color scheme, but when they allow us to use it, we can change again.